.github/workflows/cleanup-repo.yaml

```diff
@@ -1,7 +1,7 @@
 ﻿name: "Cleanup Repo"
 on:
   schedule:
```

Comment:
We could improve documentation here to explain any reasoning behind the change in schedule, so future contributors understand the context of this adjustment.